point/verb_다니까.yaml

```
@@ -0,0 +1,23 @@
+name: (느)ㄴ다니까/(이)라니까
```

```suggestion
name: (ㄴ/는)다니까/(이)라니까
```
I would go this way as the original one looks like ㄴ was mandatory no matter what.